src/shapes/Polyline.ts

```diff
@@ -169,7 +173,11 @@ export class Polyline<
       this._calcDimensions();
     this.set({ width, height, pathOffset, strokeOffset });
     adjustPosition &&
-      this.setPositionByOrigin(new Point(left, top), 'left', 'top');
+      this.setPositionByOrigin(
+        new Point(left + width / 2, top + height / 2),
```

Comment:
Could you explain was this is different than path?
Is it because the transformation is applied to the points?

Reply:
To be clear this is not different from path. Path is doing the exact same thing.